lib/unifex/code_generator.ex

```diff
@@ -22,16 +22,32 @@ defmodule Unifex.CodeGenerator do
 
   defp get_generator(%Specs{name: name, interface: nil}) do
     {:ok, bundlex_project} = Bundlex.Project.get()
+    config = bundlex_project.config
 
-    [:nifs, :cnodes]
-    |> Enum.find(&(bundlex_project.config |> Keyword.get(&1, []) |> Keyword.has_key?(name)))
-    |> case do
-      :nifs -> Unifex.CodeGenerators.NIF
-      :cnodes -> Unifex.CodeGenerators.CNode
+    type =
+      [:natives, :libs]
+      |> Enum.find(&(config |> Keyword.get(&1, []) |> Keyword.has_key?(name)))
+
+    interfaces = Keyword.get(config[type][name], :interface, [])
+
+    case interfaces do
+      [] -> Unifex.CodeGenerators.NIF
+      _ -> get_generator_module_name(List.first(interfaces))
     end
```

Comment:
I think we should raise if there's no interface at this point. Bundlex provides NIF by default for NIFs from old API, and we dropped BC for libs anyway.

Reply:
I agree but we didn't drop BC for libs at all. I mean we dropped only situation when lib without specified interface tries to use in deps other lib that has an interface but there is still support for libs without interfaces here: 

https://github.com/membraneframework/bundlex/blob/0c0e94394b8564f6ffa4f5c5355a0fcaff8f4a68/lib/bundlex/native.ex#L81-L89

Maybe we should drop lines 82 and 83 and remove support for libs without interfaces entirely

Reply:
Libs without interfaces should be supported in bundlex, but not in unifiex

Reply:
fixed